vertical-pod-autoscaler/charts/vertical-pod-autoscaler/templates/admission-controller-deployment.yaml

```diff
@@ -48,6 +48,7 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      priorityClassName: {{ .Values.admissionController.priorityClassName }}
```

Comment:
Should these be guarded with an if statement? 
It default to nil, so I think it doesn't make sense to add it if they aren't defined.